### PR TITLE
 Update try-it guide to refer v1alpha4 instead of v1alpha3

### PR DESCRIPTION
**Pull request by wgslr**
Changed mentions of v1alpha3 to v1alpha4 after verifying the scripts still work. Also fixed minor typos and added a short section about using the `config_user.sh` file

### CODE REVIEW

**Comment by fmuyassarov** [comment]
@wgslr Can you please rebase and squash the changes ? Travis issue should be fixed now.

File: try-it.md [hunk 1]
```diff
@@ -113,10 +123,10 @@ $ sudo virsh list
...
 represented by `BareMetalHost` objects in our management cluster. The yaml
-defition file used to create these host objects is in `bmhosts_crs.yaml`.
+definition file used to create these host objects is in `bmhosts_crs.yaml`.
 
 ```sh
-$ kubectl get baremetalhosts -n metal3
+$ kubectl get baremetalhosts -n metal3 -o wide
 NAME     STATUS   PROVISIONING STATUS   CONSUMER   BMC                         HARDWARE PROFILE   ONLINE   ERROR
 node-0   OK       ready                            ipmi://192.168.111.1:6230   unknown            true
 node-1   OK       ready                            ipmi://192.168.111.1:6231   unknown            true
```
Comment (fmuyassarov) [inline try-it.md:132]: Print columns have changed recently, maybe it is worth to update them too in the same PR?
```suggestion
$ kubectl get baremetalhosts -n metal3 -o wide
NAME     STATUS   STATE   CONSUMER   BMC                         HARDWARE_PROFILE   ONLINE   ERROR
node-0   OK       ready                            ipmi://192.168.111.1:6230   unknown            true
node-1   OK       ready                            ipmi://192.168.111.1:6231   unknown            true
```

**Review by fmuyassarov — COMMENTED** [review]
This looks good to me. Thank you.
I have two more suggestions that we can fix in this PR if you don't mind.
One is inline, second is CAPM3 pods aren't deployed in `capm3` namesapce anymore, but rather in `capm3-system`. Can you please fix this line too
`kubectl logs -n capm3 pod/capm3-manager-7bbc6897c7-bp2pw -c manager` ?

File: try-it.md (hunk 1 above)
Comment (wgslr) [reply try-it.md]: Done in c24db1d, please take a look and I will squash it into the previous commit.

Comment (fmuyassarov) [reply try-it.md]: @wgslr This looks good. Thanks.

Comment (fmuyassarov) [reply try-it.md]: I will lgtm once you squash the commits.

**Pull request merged**